neurom/stats.py

```diff
@@ -176,3 +176,26 @@ def compare_two(data1, data2, test=StatTests.ks):
     Stats = namedtuple('Stats', ['dist', 'pvalue'])
 
     return Stats(*results)
+
+
+def total_score(paired_dats, p=2, test=StatTests.ks):
```

Comment:
If we assume that the results can be meaningful in the first place, then p is important. The relation between the results for different p-norms should have a monotonic relation, but are not expect to be equal. How we choose an optimal p is another mystery for me...